orchestra/communication/staffing.py

```diff
@@ -39,8 +41,19 @@ def handle_staffing_response(worker, staffing_request_inquiry_id,
                 request__request=staffing_request_inquiry.request,
                 is_winner=True).exists()):
         response.is_winner = True
-        assign_task(worker.id,
-                    staffing_request_inquiry.request.task.id)
 
+        request = staffing_request_inquiry.request
+        # if task assignment exists then reassign
+        task_assignment = get_object_or_None(
+            TaskAssignment,
+            task=request.task,
+            assignment_counter=request.required_role_counter
+        )
+
+        # otherwise assign task
```

Comment:
i think the two comments are unaligned---the first one in the if, the second one in the else?